src/Fixer/Basic/CurlyBracesPositionFixer.php

```diff
@@ -354,7 +389,7 @@ protected function createConfigurationDefinition(): FixerConfigurationResolverIn
                 ->setDefault(self::SAME_LINE)
                 ->getOption(),
             (new FixerOptionBuilder('functions_opening_brace', 'The position of the opening brace of functions‘ body.'))
-                ->setAllowedValues([self::NEXT_LINE_UNLESS_NEWLINE_AT_SIGNATURE_END, self::SAME_LINE])
+                ->setAllowedValues([self::NEXT_LINE_IF_RETURN_TYPEHINT, self::NEXT_LINE_UNLESS_NEWLINE_AT_SIGNATURE_END, self::SAME_LINE])
```

Comment:
Is the style this proposal brings a part of a standard or used by a bigger (F)OSS community, eg PSR/PER-CS, Symfony, ...?

For us that is a condition to add a rule (or option) to the project.